ui/page/components/mixer_layout.go

```diff
@@ -0,0 +1,140 @@
+}
+
+func (mc MixerComponent) MixerLayout(gtx C) D {
+	r := 8
+	return cryptomaterial.LinearLayout{
+		Width:       mc.Width,
+		Height:      mc.Height,
+		Orientation: layout.Vertical,
+		Padding:     layout.UniformInset(values.MarginPadding15),
+		Background:  mc.Theme.Color.Surface,
+		Border: cryptomaterial.Border{
+			Radius: cryptomaterial.CornerRadius{
+				TopLeft:     r,
+				TopRight:    r,
+				BottomRight: r,
+				BottomLeft:  r,
+			},
+		},
+	}.Layout(gtx,
+		layout.Rigid(mc.topMixerLayout),
+		layout.Rigid(mc.middleMixerLayout),
+		layout.Rigid(mc.bottomMixerLayout),
+	)
+}
+
+func (mc MixerComponent) topMixerLayout(gtx C) D {
+	return layout.Flex{
+		Axis:      layout.Horizontal,
+		Alignment: layout.Middle,
+	}.Layout(gtx,
+		layout.Rigid(mc.Theme.Icons.Mixer.Layout24dp),
+		layout.Rigid(func(gtx C) D {
+			lbl := mc.Theme.Body1(values.String(values.StrMixerRunning))
+			lbl.Font.Weight = font.SemiBold
+			return layout.Inset{
+				Left:  values.MarginPadding8,
+				Right: values.MarginPadding8,
+			}.Layout(gtx, lbl.Layout)
+		}),
+		layout.Rigid(mc.InfoButton.Layout),
+		layout.Flexed(1, func(gtx C) D {
+			return layout.E.Layout(gtx, mc.ForwardButton.Layout)
+		}),
+	)
+}
+
+func (mc MixerComponent) middleMixerLayout(gtx C) D {
+	r := gtx.Dp(7)
+	return cryptomaterial.LinearLayout{
+		Width:       cryptomaterial.WrapContent,
+		Height:      cryptomaterial.WrapContent,
+		Orientation: layout.Horizontal,
+		Padding: layout.Inset{
+			Left:   values.MarginPadding10,
+			Right:  values.MarginPadding10,
+			Top:    values.MarginPadding4,
+			Bottom: values.MarginPadding4,
+		},
+		Margin: layout.Inset{
+			Top:    values.MarginPadding10,
+			Bottom: values.MarginPadding10,
+		},
+		Background: mc.Theme.Color.LightBlue7,
+		Alignment:  layout.Middle,
+		Border: cryptomaterial.Border{
+			Radius: cryptomaterial.CornerRadius{
+				TopLeft:     r,
+				TopRight:    r,
+				BottomRight: r,
+				BottomLeft:  r,
+			},
+		},
+	}.Layout(gtx,
+		layout.Rigid(mc.Theme.Icons.Alert.Layout20dp),
+		layout.Rigid(func(gtc C) D {
+			lbl := mc.Theme.Body2(values.String(values.StrKeepAppOpen))
+			return layout.Inset{Left: values.MarginPadding6}.Layout(gtx, lbl.Layout)
+		}),
+	)
+}
+
+func (mc MixerComponent) bottomMixerLayout(gtx C) D {
+	r := 8
+	return cryptomaterial.LinearLayout{
+		Width:       cryptomaterial.WrapContent,
+		Height:      cryptomaterial.WrapContent,
+		Orientation: layout.Vertical,
+		Padding:     layout.UniformInset(values.MarginPadding15),
+		Background:  mc.Theme.Color.Gray4,
+		Border: cryptomaterial.Border{
+			Radius: cryptomaterial.CornerRadius{
+				TopLeft:     r,
+				TopRight:    r,
+				BottomRight: r,
+				BottomLeft:  r,
+			},
+		},
+	}.Layout(gtx,
+		layout.Rigid(func(gtc C) D {
+			lbl := mc.Theme.Body2(mc.WalletName)
+			lbl.Font.Weight = font.SemiBold
+			return lbl.Layout(gtx)
+		}),
+		layout.Rigid(func(gtx C) D {
+			return layout.Flex{
+				Axis:      layout.Horizontal,
```

Comment:
don't need this line because default is Horizontal

Reply:
same as above.